src/main/java/xbot/common/subsystems/drive/SwerveCalculatorNode.java

```diff
@@ -0,0 +1,12 @@
+package xbot.common.subsystems.drive;
+
+
+/**
+ * @param operationEndingVelocity m/s, velocity we end at
+ * @param operationAcceleration   mls^2, acceleration to go at
+ * @param operationTime           Time in seconds
+ */ // SwerveCalculatorNode are small little steps to be interpreted and formed together
```

Comment:
⭐ consider moving this explanation up into the javadoc comment above, that way it'll appear when people hover over this class in their IDE. https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html